autode/conformers/conformer.py

```diff
@@ -41,6 +41,7 @@ def __init__(
             self.atoms = Atoms(atoms)
 
         self.constraints.update(distance=dist_consts)
+        self.boltzmann_weight = None
```

Comment:
does this want to be settable by users? feels like maybe not. I would have this as a private attribute `self._boltzmann_weight` and define a getter

```python
@propery
def boltzmann_weight() -> Optional[float]:
     """docstring"""
    return self._boltzmann_weight
```

also means that theres a nice docstring explaining what it is and what type (i.e. None or float) will be returned